server/gokbg3/grails-app/views/apptemplates/_tipp.gsp

```diff
@@ -62,6 +62,13 @@
                             config='KBComponent.EditStatus'/>
     </dd>
 
+    <dt>
+        <g:annotatedLabel owner="${d}" property="language">Language</g:annotatedLabel>
+    </dt>
+    <dd>
+        <g:xEditableRefData owner="${d}" field="language" config='KBComponent.Language'/>
```

Comment:
All other templates are extracting the category name from KBComponent directly, should be done consistently.